### ISSUE TITLE
[Bug]: Frameless window gets immovable by dragging

### ISSUE THREAD

**Issue by angkor-wat** [electron/electron#33443]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/electron/electron/issues) for a bug report that matches the one I want to file, without success.

### Electron Version

17.1.2

### What operating system are you using?

Windows

### Operating System Version

Windows 10 Family, 10.0.19043

### What arch are you using?

x64

### Last Known Working Electron version

none

### Expected Behavior

```
new BrowserWindow({
 show: false,
 frame: false,
 titleBarStyle: 'hidden',
 titleBarOverlay: {
   color: '#2f3241',
   symbolColor: '#74b1be'
 },
 minWidth: 450,
 minHeight: 500,
 icon: app.getAppPath() + '/logo/256.ico',
 title: app.getName(),
 useContentSize: true,
 webPreferences: {
   devTools: global.isDebug,
   nodeIntegration: true,
   contextIsolation: false,
   preload: path.join(app.getAppPath(), 'render/common','preload.js')
 }
 })
```

```
body{
    app-region: drag;
    user-select: none;
    -webkit-app-region: drag;
    -webkit-user-select: none;
}
```

Expected: A Fameless window with overlay titleBar can be moved and resized.

### Actual Behavior

The window can not be moved.

**Comment by codebytere** [comment]
Thanks for reporting this and helping to make Electron better!

Because of time constraints, triaging code with third-party dependencies is usually not feasible for a small team like Electron's.

Would it be possible for you to make a standalone testcase with only the code necessary to reproduce the issue? For example, [Electron Fiddle](https://www.electronjs.org/fiddle) is a great tool for making small test cases and makes it easy to publish your test case to a [gist](https://gist.github.com) that Electron maintainers can use.

Stand-alone test cases make fixing issues go more smoothly: it ensure everyone's looking at the same issue, it removes all unnecessary variables from the equation, and it can also provide the basis for automated regression tests.

I'm adding the `blocked/need-repro` label for this reason. After you make a test case, please link to it in a followup comment. This issue will be closed in 10 days if the above is not addressed.

**Comment by Haeri** [comment]
The issue seems to be resolved with electron 17.3.0 and higher
